### PR TITLE
Implement deny assignment

### PR DESCRIPTION
**Pull request by jim-minter**
Builds on #381

### CODE REVIEW

**Comment by jim-minter** [comment]
Pretty much ready, but don't merge yet.

File: pkg/install/0-installstorage.go [hunk 1]
```diff
@@ -82,10 +88,45 @@ func (i *Installer) installStorage(ctx context.Context, installConfig *installco
...
 		Schema:         "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
 		ContentVersion: "1.0.0.0",
 		Resources: []*arm.Resource{
+			{
+				Resource: &mgmtauthorization.RoleAssignment{
+					Name: to.StringPtr("[guid(resourceGroup().id, 'SP / Contributor')]"),
+					Type: to.StringPtr("Microsoft.Authorization/roleAssignments"),
+					RoleAssignmentPropertiesWithScope: &mgmtauthorization.RoleAssignmentPropertiesWithScope{
+						Scope:            to.StringPtr("[resourceGroup().id]"),
+						RoleDefinitionID: to.StringPtr("[resourceId('Microsoft.Authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c')]"),
```
Comment (mjudeikis) [inline pkg/install/0-installstorage.go:123]: Comment on what is this `UUID`

File: pkg/install/0-installstorage.go [hunk 2]
```diff
@@ -82,10 +88,45 @@ func (i *Installer) installStorage(ctx context.Context, installConfig *installco
 		}
 	}
 
+	var objectID string
```
Comment (mjudeikis) [inline pkg/install/0-installstorage.go:91]: `clusterSPID`?

**Comment by jim-minter** [comment]
Closing for #394.  Changes requested here have been applied there.

**Pull request closed without merging**